Projects/App/Sources/Presentation/MainHome/MainHomeViewController.swift

```diff
@@ -0,0 +1,317 @@
+        dimView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+        dimView.isHidden = true
+    }
+    
+    private func applyData(items: [MeetingMain], section: Section) {
+        var snapshot = NSDiffableDataSourceSnapshot<Section, MeetingMain>()
+        snapshot.appendSections([section])
+        snapshot.appendItems(items)
+        self.dataSource.apply(snapshot, animatingDifferences: true)
+        self.collectionView.setContentOffset(.init(x: -collectionView.contentInset.left, y: .zero), animated: true)
+    }
+    
+    private func setAction() {
+        proceedingPromiseButton.rx.tap
+            .asDriver()
+            .drive(onNext: { [weak self] in
+                // 진행중인 약속
+                self?.proceedingPromiseButton.isEnabled = false
+                self?.endedPromiseButton.isEnabled = true
+                let proceeding = self?.fetchedMainHomeMeeting.proceedingMeetings ?? []
+                self?.applyData(items: proceeding, section: .proceeding)
+                
+            }).disposed(by: bag)
+        
+        endedPromiseButton.rx.tap
+            .asDriver()
+            .drive(onNext: { [weak self] in
+                // 종료된 약속
+                self?.proceedingPromiseButton.isEnabled = true
+                self?.endedPromiseButton.isEnabled = false
+                
+                let ended = self?.fetchedMainHomeMeeting.endedMeetings ?? []
+                self?.applyData(items: ended, section: .ended)
+            }).disposed(by: bag)
+        
+        floatingButton.rx.tap
+            .asDriver()
+            .drive(onNext: { [weak self] in
+                self?.dimView.isHidden = false
+            }).disposed(by: bag)
+        
+        dimView.floatingButton.rx.tap
+            .asDriver()
+            .drive(onNext: { [weak self] in
+                self?.dimView.isHidden = true
+            }).disposed(by: bag)
+        
+        dimView.addPromiseButton.rx.tap
+            .asDriver()
+            .drive(onNext: { [weak self] in
+                let vm = RoomNameViewModel(joinAdminUseCase: JoinHostUseCase())
+                self?.navigationController?.pushViewController(RoomNameViewController(viewModel: vm), animated: true)
+            }).disposed(by: bag)
+        
+        dimView.enterWithCodeButton.rx.tap
+            .asDriver()
+            .drive(onNext: { [weak self] in
+                let vm = RoomCodeViewModel(joinGuestUseCase: JoinGuestUseCase())
+                self?.navigationController?.pushViewController(RoomCodeController(viewModel: vm), animated: true)
+            }).disposed(by: bag)
+    }
+    
+    private func bindRx() {
+        let input = MainHomeViewModel.Input(viewDidLoad: Observable.just(()).asObservable())
+        
+        let output = viewModel?.transform(input: input, disposeBag: bag)
+        
+        output?.mainHomeMeeting.subscribe(onSuccess: { [weak self] in
+            print("mainHomeMeeting",$0)
```

Comment:
p5) 
개발중이라 상관은 없지만 배포코드에는 안들어가도록 주석이나 debugprint 로 바꾸기 나중에 잊지말기 노티용 리뷰!
```suggestion
        output?.mainHomeMeeting.subscribe(onSuccess: { [weak self] in
        debugPrint("mainHomeMeeting",$0)
```

Reply:
알게쑴다!